keyboards/3.0/bn.xml

```diff
@@ -0,0 +1,155 @@
+        <import base="cldr" path="techpreview/keys-Zyyy-currency.xml" />
+
+        <!-- key names based on https://www.loc.gov/catdir/cpso/romanization/bengali.pdf + Assamese for /wa/ -->
+
+        <!-- UNSHIFTED KEYS -->
+        <!-- E: (top) row -->
+        <key id="n̐" output="\u{0981}" /> <!-- candrabindu-->
+        <key id="1" output="১" />
+        <key id="2" output="২" />
+        <key id="3" output="৩" />
+        <key id="4" output="৪" />
+        <key id="5" output="৫" />
+        <key id="6" output="৬" />
+        <key id="7" output="৭" />
+        <key id="8" output="৮" />
+        <key id="9" output="৯" />
+        <key id="0" output="০" />
+        <!-- hyphen -->
+        <!-- equal -->
+
+        <!-- D: -->
+        <key id="more" output="\m{q}" /> <!-- 'additional characters' -->
+        <key id="wa" output="ৱ" /> <!-- Assamese-->
+        <key id="e" output="\u{09C7}" />
+        <key id="ra" output="র" />
+        <key id="ta" output="ত" />
+        <key id="sha" output="য" />
+        <key id="u" output="\u{09C1}" />
+        <key id="i" output="\u{09BF}" />
+        <key id="o" output="\u{09CB}" />
+        <key id="pa" output="প" />
+
+        <!-- C: -->
+        <key id="ā" output="\u{09BE}" />
+        <key id="sa" output="স" />
+        <key id="da" output="দ" />
+        <key id="ṭa" output="ট" />
+        <key id="ga" output="গ" />
+        <key id="ha" output="হ" />
+        <key id="ja" output="জ" />
+        <key id="ka" output="ক" />
+        <key id="la" output="ল" />
+
+        <!-- B: -->
+        <key id="śa" output="শ" />
+        <key id="hasant" output="\u{09CD}" />
+        <key id="ca" output="চ" />
+        <key id="ḍa" output="ড" />
+        <key id="ba" output="ব" />
+        <key id="na" output="ন" />
+        <key id="ma" output="ম" />
+        <!-- comma -->
+        <key id="dahri" output="।" />
+
+
+        <!-- SHIFTED KEYS -->
+        <key id="ṃ" output="\u{0982}" />
+        <!-- exclam, etc -->
+
+        <!-- gap -->
+        <key id="ña" output="ঞ" />
+        <key id="ai" output="\u{09C8}" />
+        <key id="ṛ" output="\u{09C3}" />
+        <key id="tha" output="থ" />
+        <key id="ẏa" output="য়" /> <!-- Missing in Keyman version of file-->
+        <key id="ū" output="\u{09C2}" />
+        <key id="ī" output="\u{09C0}" />
+        <key id="au" output="\u{09CC}" />
+        <key id="pha" output="ফ" />
+
+        <key id="au-length" output="\u{09D7}" /> <!-- TODO: better name? -->
+        <key id="sha" output="ষ" />
+        <key id="dha" output="ধ" />
+        <key id="ṭha" output="ঠ" />
+        <key id="gha" output="ঘ" />
+        <!-- gap -->
+        <key id="jha" output="ঝ" />
+        <key id="kha" output="খ" />
+        <!-- gap -->
+
+        <!-- gap -->
+        <key id="vis-hasant" output="\m{X}" />
+        <key id="cha" output="ছ" />
+        <key id="ḍha" output="ঢ" />
+        <key id="bha" output="ভ" />
+        <key id="ṅa" output="ণ" />
```

Comment:
both ALA-LC and ISO15959 say this is ṇa

Reply:
thanks good catch